packages/next/next-server/lib/post-process.ts

```diff
@@ -12,48 +17,59 @@ type postProcessData = {
   }
 }
 
-type postProcessMiddleware = (
-  htmlRoot: HTMLElement,
-  data: postProcessData
-) => Promise<void>
+interface PostProcessMiddleware {
```

Comment:
@atcastle the intent of the change is to make it explicit that changes made to `originalDom` in `inspect` wont be affected in the final dom. You'll need to  do string manipulation in `mutate` for actual changes